config/config.default.json

```diff
@@ -3,6 +3,8 @@
   "basePath": "http://localhost:3050",
   "matrixServerUrl": "http://localhost:8008/",
   "archiveMessageLimit": 500,
+  "requestTimeoutMs": 25000,
+  "UNCOMMENT_jaegerTracesEndpoint": "http://localhost:14268/api/traces",
```

Comment:
Would be good to introduce `jsonc` to allow comments in the config.

I would like to display this config option without having it on by default since it requires a third-party service (Jaeger).